pandas/tests/scalar/timestamp/test_comparisons.py

```diff
@@ -104,7 +103,7 @@ def test_cant_compare_tz_naive_w_aware(self):
         pytest.raises(Exception, b.__lt__, a)
         pytest.raises(Exception, b.__gt__, a)
 
-        if sys.version_info < (3, 3):
+        if PY2:
```

Comment:
Likewise, <3.3 implies `PY2` based on the currently supported versions.  Could be more explicit if desired though.

Reply:
That's reasonable.  2.7, 3.5 - 3.6 is the way to go.